cookbook/external_completers.md

```diff
@@ -134,20 +136,27 @@ let carapace_completer = {|spans: list<string>|
 
 # This completer will use carapace by default
 let external_completer = {|spans|
-    let expanded_alias = (scope aliases | where name == $spans.0 | get -i 0 | get -i expansion)
-    let spans = (if $expanded_alias != null  {
-        $spans | skip 1 | prepend ($expanded_alias | split words)
-    } else { $spans })
+    let expanded_alias = scope alias-completions
+    | where name == $spans.0
+    | get -i 0.expansion
+
+    let spans = if $expanded_alias != null {
+        $spans
+        | skip 1
+        | prepend ($expanded_alias | split row ' ')
+    } else {
+        $spans
+    }
 
-    {
+    match {
```

Comment:
What is `match` matching on here?

Reply:
Oops, that's a typo. It should be `match $spans.0` :)

I'll create a followup PR soon when I have time